documenting normalizePath and update submission-guidelines (#12)
marcusolsson/obsidian-plugin-docs: docs/publishing/submit-your-plugin.md

1. In `manifest.json`, update `version` to a new version that follows the [Semantic Versioning](https://semver.org/) specification.

> 1. [Create a GitHub release](https://docs.github.com/en/repositories/releasing-projects-on-github/managing-releases-in-a-repository#creating-a-release). The  "Tag version" of the release must match the version in your `manifest.json`. Don't include a `v` in the tag version. It is also good practice to provide a release title and to describe the release in the description field. 

Reviewer: ```suggestion
1. [Create a GitHub release](https://docs.github.com/en/repositories/releasing-projects-on-github/managing-releases-in-a-repository#creating-a-release). 
   - The  "Tag version" of the release must match the version in your `manifest.json`. 
   - Don't include a `v` in the tag version. 
1. Enter a name for the release, and describe it in the description field.
```

Good catch!

- If it's a good practice, we should ask the reader to do it.
- For text fields, the Microsoft style guide uses "Enter" rather than "Provide".

   ```

   - `id`, `name`, `author`, and `description` determines how your plugin appears to the user, and should match the corresponding properties in your [plugin manifest](manifest-reference.md).

>    - `id` should be unique to your plugin and unique in the `community-plugins.json`. Therefore do a search to confirm there is no conflict.

Reviewer: ```suggestion
   - `id` is unique to your plugin. Search `community-plugins.json` to confirm that there's no existing plugin with the same id.
```

- "Should" implies that there are cases where it's ok.
- "Do a search" is a [nominalization](https://en.wikipedia.org/wiki/Nominalization), prefer "Search".
